foundation-extensions/identity-for-edge-network/README.md

```diff
@@ -145,3 +165,24 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 ```
 {% endtab %}
 {% endtabs %}
+
+## Advertising Identifier
+The Edge Identity extension compares the previously stored advertising identifier value with the new value received from the `setAdvertisingIdentifier` API and handles the following scenarios:
+
+Ad tracking enabled - when the new value sent to the API is:
+- A valid UUID string (example: `"a127a99e-50be-4d87-bf6f-6ab9541c105b"`)
+
+Process:
+1. Updates the client side XDM `IdentityMap` with the new value for IDFA/GAID, and it will be sent with any future [XDM Experience events](../experience-platform-extension/xdm-experience-events.md). For more details on these namespaces see the [standard Identity namespaces](https://experienceleague.adobe.com/docs/experience-platform/identity/namespaces.html#standard). 
```

Comment:
Updates the client-side XDM `IdentityMap` with the new value for IDFA/GAID, which is included in subsequent [XDM Experience events](../experience-platform-extension/xdm-experience-events.md). For more details, see the ...
- something similar for the remove use-case